src/lib/packet.h

```diff
@@ -355,6 +355,7 @@ typedef enum {
     PGP_PKA_RESERVED_DH = 21,             /* Reserved for Diffie-Hellman
                                            * (X9.42, as defined for
                                            * IETF-S/MIME) */
+    PGP_PKA_EDDSA = 22,                   /* EdDSA */
```

Comment:
@catap the constant of `22` is also defined here: https://tools.ietf.org/html/draft-ietf-openpgp-rfc4880bis-01#section-9.1

I agree that we can add a comment about this.